src/App.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "./App.css";
+import Header from "./Header";
+import HeroesGrid from "./HeroesGrid";
+
+class App extends React.Component {
+  state = {
+    heroes: []
+  };
+
+  componentDidMount() {
+    this.getRequestInJSON("https://api.opendota.com/api/heroStats").then(
+      heroesGot => {
+        this.setState({
+          heroes: heroesGot,
+          allHeroes: heroesGot
+        });
+      }
+    );
+  }
+
+  getRequestInJSON = url => {
+    return fetch(url).then(response => response.json());
+  };
+
+  filterHeroes = heroSearched => {
+    if (heroSearched === "") {
+      this.setState(prevState => {
+        return { heroes: prevState.allHeroes };
+      });
+    } else {
+      this.setState(prevState => {
+        return {
+          heroes: prevState.allHeroes.filter(function(elem, index, array) {
```

Comment:
you used arrow function everywhere, but here 👀
Another way to check if the string is inside another:
``` javascript
elem.localized_name.toLowerCase().includes(heroSearched.toLowerCase())
```
[includes method link](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/String/includes)
This way you can find "anti-mage" by search: "mage"